curator-client/src/test/java/org/apache/curator/utils/TestZKPaths.java

```diff
@@ -26,6 +26,18 @@
 
 public class TestZKPaths
 {
+    @Test
```

Comment:
I'm +1 on this change but do you mind writing more tests in `TestZKPaths`? I'd like to see more complete coverage of the new methods.

Reply:
@marquiswang any chance for more tests?

Reply:
Sorry, I have been on an extended trip and have not had a chance to work on this. I can definitely add tests when I get home in mid March.

Reply:
ping @marquiswang do you have some spare time to bring back this patch? You can merge master and add tests :)

Reply:
Hello, blast from the past. I supppooooose I can look into it when I get a moment.

Reply:
@marquiswang lol. Go ahead!